server/src/internal_backstage.rs

```diff
@@ -6,8 +6,10 @@ use actix_web::{
     get,
     web::{self, Json},
 };
-use serde::Serialize;
-#[derive(Debug, Serialize)]
+use dashmap::DashMap;
+use serde::{Deserialize, Serialize};
+use unleash_types::client_features::ClientFeatures;
+#[derive(Debug, Serialize, Deserialize)]
 pub struct EdgeStatus {
```

Comment:
Might be time for the inner type to be an enum here. Should be okay to do something like 

enum Status {
  Ok,
  Ready,
  NotReady
}

Small suggestion, I'm okay for this to go without it